lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_node_lowering.cpp

```diff
@@ -319,9 +319,17 @@ class TSNodeLowering : public TSNodeLoweringInterface {
             ? LowerBuiltin(at::aten::cudnn_convolution_transpose_backward,
                            arguments)
             : LowerBuiltin(at::aten::cudnn_convolution_backward, arguments);
-    // cudnn_convolution_backward/cudnn_convolution_transpose_backward only
-    // returns 2 tensors
-    result.push_back(nullptr);
+
+    // N.B. rank is specialized even with dynamic shapes
+    // all axes but channel to reduce to the bias size
+    auto grad_rank = dynamic_cast<const ir::TsNode*>(operands[0].node)->shape(operands[0].index).dim();
```

Comment:
better to use GetShapeFromTsNode or GetShapeFromTsValue helpers here, just so we can easily refactor them away or grep them later